### DISCUSSION TITLE
Keep open a dialog opened from a dropdown?

### DISCUSSION THREAD

**Discussion by abnormalyak** [quasarframework/quasar#11628] [General - Components / Directives / etc] [answered]
Given a q-dialog, opened from a button within a button dropdown, interacting with the dialog will close it instantly due to it being spawned from the dropdown. This is because the dropdown creates temporary HTML elements that are removed after it is closed, and the dialog is opened within that. This can be solved by making the dropdown "persistent", but that means that after the user is finished interacting with the dialog and closes it, the dropdown itself stays open.

Is there a solution where the dropdown can close and the dialog stays open, or should I instead close the dropdown when the dialog closes? (or any other solution is welcome!)

Thank you for any input!

**Comment by pdanpdan** [comment, marked as answer]
Use a dialog plugin.
Check the custom component section in dialog plugin

**Reply by abnormalyak** [reply]
Thank you, I'll look into that. Is there a way to close a q-dropdown when receiving a custom event from a component?

**Comment by pdanpdan** [comment]
Use v-model and set it to false when you want to close

**Reply by abnormalyak** [reply]
Thank you very much for your help!
